### DISCUSSION TITLE
Removing automatic grammar correction in Whisper

### DISCUSSION THREAD

**Discussion by Shikhs1989** [openai/whisper#1631] [Q&A]
Whisper is correcting grammar in the transcription. How can we achieve raw output in transctiption. Also, due to this it is not transcribing one word transcriptions correctly. E.g It is converting 'chemist' to 'i missed.'.

**Comment by whicks1** [comment]
There is a built in normalizer that you can’t do much about without probably forking the project. It is described in the paper and is fairly readable in the code but in short it removes “ums”, standardizes contractions and various numbers, etc. Additionally, I would suspect that since it’s fairly standard practice for human transcribers to edit outputs and vendors charge extra for verbatim outputs, a significant portion of the training data is “cleaned” and so whisper will probably follow those patterns as well. As with all other problems someone will suggest you attempt to use the initial prompt argument but your milage will vary. You might also look at setting the temp. Finally as with any asr, the outputs aren’t perfect and spelling and interpretation errors are going to happen, use the large model of you can. Otherwise, review the outputs and post process ftw.

**Reply by Shikhs1989** [reply]
Hi Whicks1, Thanks for replying. 
Is there any parameter that we can disable to stop the corrections and it should just give raw data as provided in the audio?

**Reply by whicks1** [reply]
I think the simple answer is no.

**Reply by this-duck** [reply]
Could you perhaps explain how adjusting the temprature would assist?
